code/game/objects/structures/crates_lockers/closets/__closet.dm

```diff
@@ -55,6 +55,12 @@ var/global/list/closets = list()
 	if(!opened && mapload) // if closed and it's the map loading phase, relevant items at the crate's loc are put in the contents
 		store_contents()
 
+/obj/structure/closet/update_lock_overlay()
+	return // TODO
+
+/obj/structure/closet/can_install_lock()
```

Comment:
Requesting `/obj/structure/closet/body_bag/can_install_lock() return FALSE` in `bodybag.dm` to avoid installing a lock on what amounts to a large plastic bag. Also unsure if you should be able to put one on `/obj/structure/closet/secure_closet` since it already has a digital lock.